content/auth/capabilities.textile

```diff
@@ -191,4 +191,25 @@ const fromModerator = (message) => {
     const userClaim = message.extras && message.extras.userClaim;
     return (userClaim && userClaim == 'moderator');
 }
-```
\ No newline at end of file
+```
+
+h2(#jwt-limits). Using JWT for per connection publish rate limits
+
```

Comment:
I don't see anything that specifies the unit of time for the setting. It can be deduced from the code example, but not entirely clear. Consider stating in the text that the limit is specified as the number of messages that can be published per second in the channel, and explain how numbers smaller than 1 are treated.

Reply:
I've updated with this info 👍